src/poliastro/twobody/events.py

```diff
@@ -80,3 +87,80 @@ class LithobrakeEvent(AltitudeCrossEvent):
 
     def __init__(self, R, terminal=True):
         super().__init__(0, R, terminal, direction=-1)
+
+
+class LatitudeCrossEvent(Event):
+    """Detect if a satellite crosses a specific threshold latitude.
+
+    Parameters
+    ----------
+    lat: float
+        Threshold latitude (km).
+    R: float
+        Radius of the attractor (km).
+    terminal: bool
+        Whether to terminate integration if this event occurs, defaults to True.
+    direction: float
+        Handle triggering of event based on whether latitude is crossed from above
+        or below, defaults to 0, i.e., event is triggered while traversing from both directions.
+
+    """
+
+    def __init__(self, lat, R, terminal=True, direction=0):
+        super().__init__(terminal, direction)
+        self._R = R
+        self._lat = lat  # Threshold latitude (in degrees).
+
+    def __call__(self, t, u_, k):
+        self._last_t = t
+        xyz = u_[:3]
+
+        obstime = Time(self._last_t, format="jd")
```

Comment:
The `t` variable is just a float value representing the amount of seconds since the initial orbit epoch. Therefore, we require to store the initial orbit epoch to build the right epoch value after `t` elapsed seconds:

```python
Time(self._last_t = self._epoch0 + t * u.s)
```